src/fmpz_poly/test/t-eulerian_polynomial.c

```diff
@@ -23,59 +23,55 @@ TEST_FUNCTION_START(fmpz_poly_eulerian_polynomial, state)
     fmpz_init(sum);
     fmpz_init(fac);
 
-    for (ix = 0; ix < 100; ix++)
+    for (ix = 0; ix < 1000 * flint_test_multiplier(); ix++)
     {
-        n = n_randint(state, 1000) + 1; /* Don't want n = 0. */
+        n = 1 + n_randint(state, 200);
```

Comment:
By the pigeonhole principle, it is a bit silly to run a much larger number of random test iterations than the possible number of distinct n being tested...

Reply:
Fixing old bugs and introducing new ones :sunglasses: